src/app/modules/entry-live/views/live-stream-health/live-stream-health.component.ts

```diff
@@ -64,7 +82,24 @@ export class LiveStreamHealthComponent implements OnInit, OnDestroy {
       }
       return 0;
     };
-    
-    return [...response.streamHealth.data.primary, ...response.streamHealth.data.secondary].sort(sortHealthNotifications);
+
+    this._isSelfServe = response.selfServe.data;
```

Comment:
@adimachmali - do we always get response.selfServe? If not - we might encounter a runtime error. In that case better change to:
`this._isSelfServe = response.selfServe && response.selfServe.data;`

Reply:
Basically there is an initial value at:
https://github.com/kaltura/analytics-front-end/blob/400a6f868ace3b240edbf82fe5174e8e1d377752/src/app/modules/entry-live/views/live-stream-health/live-stream-health.widget.ts#L33

But I will add this small check, to be on the safe side.